src/cpp/session/modules/build/SessionBuild.cpp

```diff
@@ -1077,7 +1077,8 @@ class Build : boost::noncopyable,
           !options_.autoRoxygenizeForCheck)
          args.push_back("document = FALSE");
 
-      if (!prefs::userPrefs().cleanupAfterRCmdCheck())
+      if (!prefs::userPrefs().cleanupAfterRCmdCheck() && 
+          !module_context::isPackageVersionInstalled("devtools", "2.4.4"))
```

Comment:
Assuming there is no need to check that `devtools` is indeed installed here, otherwise we wouldn't be here.

Reply:
I think you're right about that